app/celery/send_va_profile_notification_status_tasks.py

```diff
@@ -0,0 +1,76 @@
+import requests
```

Comment:
Moved from `app/celery/process_ses_receipts_tasks.py`. 
Methods renames to denote they handle both SMS and Email notifications.
Added logic to continue sending Email notifications even if `VA_PROFILE_SMS_STATUS_ENABLED` disabled.